app/build.gradle

```diff
@@ -109,6 +107,7 @@ android {
     configurations.all {
         resolutionStrategy.force "com.android.support:support-annotations:$versions.support"
     }
+    buildToolsVersion '26.0.2'
```

Comment:
Duplication with line 19 (fix [deps.gradle](https://github.com/fs/android-base/blob/master/deps.gradle))

Reply:
Need update the version for ci